Commencement.Mvc/Controllers/TicketController.cs

```diff
@@ -16,12 +16,18 @@ namespace Commencement.Controllers
     [Ticketing]
     public class TicketController : ApplicationController
     {
+        private readonly ICeremonyService _ceremonyService;
         private readonly string _serverLocation = ConfigurationManager.AppSettings["ReportServer"];
 
+        public TicketController(ICeremonyService ceremonyService)
+        {
+            _ceremonyService = ceremonyService;
+        }
+
         public ActionResult Index()
         {
-            var term = TermService.GetCurrent();
-            var ceremonies = term.Ceremonies;
+            //var xxx = term.Ceremonies.ToList(); //Scott, this is generating a Lazy Load error

```

Comment:
@srkirkland Not sure why this wasn't working, but it wasn't the correct way to do it anyway.